src/main/java/com/bt/code/egress/App.java

```diff
@@ -82,9 +91,29 @@ public void run(ApplicationArguments args) {
 
         wordReplacer.saveGenerated(writeGeneratedReplacement.toPath());
 
+        logErrors(fileReplacer);
```

Comment:
can it be moved to fileReplacer? so in App only something like ``fileReplacer.verify()``

Reply:
Moved

Reply:
Sorry, missed that. Method removed